core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
```

Comment:
You do not need to add any additional classes anywhere because we can reuse an existing one, `.oppia-footer-container`. You can replace `.oppia-footer-margin` inside the media query with `.oppia-footer-container` and reduce the padding.

This means that you can delete the `oppia-footer-margin` classes that you did in the css and html.